src/abyssal_tome/views.py

```
@@ -0,0 +1,18 @@
+class CardView:
+    def display_card(self, card) -> None: """
```

Ruff (0.11.9)

Missing type annotation for function argument `card`

(ANN001)